manifests/init.pp

```diff
@@ -28,7 +29,8 @@
   }
 
   class { "rpmbuilder::packages::essential":
-    epel  => $add_epel,
+    epel         => $add_epel,
+    mock_version => $mock_version,
```

Comment:
So if `mock_version` is `undef`, it won't override the default in `rpmbuilder::packages::essential`?